paasd/tests/integration.rs

```diff
@@ -0,0 +1,87 @@
+use std::net::SocketAddr;
+use std::sync::Once;
+
+use tokio;
+use tonic::Code;
+use uuid::Uuid;
+
+use paas_types::{ExecRequest, StatusRequest};
+use paasc::make_client;
+use paasd::make_server;
+
+fn exec_request(args: &[&str]) -> ExecRequest {
+    ExecRequest {
+        args: args.iter().cloned().map(ToOwned::to_owned).collect(),
```

Comment:
I don't think `cloned` is needed here.

Reply:
`iter()` on a slice of `&str` results in an iterator of `&&str`, clones here are to get back to `&str`. Perhaps `copied()` would better signal the intent.

Is there some cleaner way I'm not aware of to not clone here?

Reply:
I believe copied would be better here. It's that or doing another deref in the map closure, whichever you think is cleaner.